### DISCUSSION TITLE
Custom AR functionality development

### DISCUSSION THREAD

**Discussion by QiteBlock** [google/model-viewer#2046] [Ideas]
Hello, 

I'm wondering if it's possible to develop custom AR functionality ? 
For example, some AR app can detect foot and wear basket on it.

**Comment by elalish** [comment]
Hmm, I think that's beyond the simple use cases model-viewer is designed for. I think you'll need the WebXR layer directly for this kind of thing, though it won't be easy there either yet. You're welcome to fork model-viewer and give it a shot.

**Reply by QiteBlock** [reply]
Thanks, i understand. I will in the future try it.
